.travis.yml

```diff
@@ -20,10 +19,11 @@ addons:
    postgresql: "9.4"
 
 before_script:
-   - psql -c "CREATE ROLE full_access;" -U postgres
+   # Used by (so far non-existent) spec tests for xmlforms
```

Comment:
Fair point about local/Travis differentiation. I'll look more closely into where `initialisePostgresql()` is called, somehow I missed that function when tracing code yesterday, so I probably missed other things as well.